### PR TITLE
runs autopep8 on all python code

### PR DESCRIPTION
**Pull request by merlinran**
This simply runs `autopep8 **/*.py --in-place` from the root directory. For Atom, there's a plugin https://atom.io/packages/python-autopep8 . Further python refactors will be based off this branch.

To make things simpler, I managed to resist the temptation of doing any customization, esp. loosening the max line length of 80.

### CODE REVIEW

File: server/system_manager.py [hunk 1]
```diff
@@ -34,8 +36,6 @@
 from server import active_site
 # Necessary so pickle can access class definitions from vehicle.
 sys.path.append('../vehicle')
```
Comment (merlinran) [inline server/system_manager.py:38]: Re-ordering like this do break the code, and the formatter keeps doing. In a follow up PR I added the `/acorn/vehicle` directory to `PYTHONPATH` so the server can be up, but for scripts in `reporting/` and `bringup/` etc to work, `PYTHONPATH` needs to be defined somewhere on RPi, preferably `/etc/profile` or `/etc/profile.d/....sh`. It's unclear to me where's the best place to do that, `bringup/bringup.sh`?

File: server/system_manager.py [hunk 2]
```diff
@@ -92,38 +95,38 @@ def main():
...
+                        redis_utils.clear_autonomy_hold(
+                            redis_client=r, vehicle_name=robot.name, value=True, active_site=active_site)
                         time.sleep(_LONG_PAUSE_SEC)
                         continue
                     if AUTONOMY_AT_STARTUP:
                         if robot.control_state == CONTROL_ONLINE:
                             print("Activating Autonomy")
-                            redis_utils.set_vehicle_autonomy(redis_client=r, vehicle_name=robot.name, speed=float(AUTONOMY_SPEED), enable=True, active_site=active_site)
+                            redis_utils.set_vehicle_autonomy(redis_client=r, vehicle_name=robot.name, speed=float(
+                                AUTONOMY_SPEED), enable=True, active_site=active_site)
```
Comment (merlinran) [inline server/system_manager.py:111]: The formatter does its best, but such line breaks are not optimal - manual tune is required as drive-by of other changes.

**Comment by merlinran** [comment]
@tlalexander do you think it's possible to merge this? With all the re-formatting happening, it's nearly impossible to make comments on the important changes of https://github.com/Twisted-Fields/acorn-precision-farming-rover/pull/3. Alternatively, if you are not confident enough, this can be merged to a new branch, then I can set the new branch as the base of https://github.com/Twisted-Fields/acorn-precision-farming-rover/pull/3.

**Comment by tlalexander** [comment]
If it runs in simulation for you that is good enough I think. I did not test this but I wanted to push it forward. I assume it is working for you.

**Comment by merlinran** [comment]
Thanks! I'll go ahead to add comments to the next PR.
